src/clean_cnpj_info_dataset.py

```diff
@@ -0,0 +1,78 @@
+from itertools import chain
+import json
+import numpy as np
+import pandas as pd
+
+def decompose_main_activity(value):
+    struct = json.loads(value.replace('\'', '"'))
+    return pd.Series([*struct]). \
+        rename_axis({'code': 'main_activity_code', 'text': 'main_activity'})
+
+def decompose_secondary_activities(value):
+    struct = json.loads(value.replace('\'', '"'))
+    if bool(struct) and struct[0]['text'] != 'Não informada':
+        new_attributes = [pd.Series(activity). \
+            rename_axis({'code': 'secondary_activity_%i_code' % (index + 1),
+                         'text': 'secondary_activity_%i' % (index + 1)})
+            for index, activity in enumerate(struct)]
+        return pd.concat(new_attributes)
+    else:
+        return pd.Series()
+
+
+
+data = pd.read_csv('data/cnpj_info.xz',
+                   dtype={'atividade_principal': np.str,
+                          'atividades_secundarias': np.str,
+                          'complemento': np.str,
+                          'efr': np.str,
+                          'email': np.str,
+                          'message': np.str,
+                          'motivo_situacao': np.str,
+                          'situacao_especial': np.str})
+data = data.drop_duplicates('cnpj')
+data.rename(columns={
+    'abertura': 'opening',
+    'atividade_principal': 'main_activity',
+    'atividades_secundarias': 'secondary_activities',
+    'bairro': 'neighborhood',
+    'cep': 'zip_code',
+    'complemento': 'additional_address_details',
+    'data_situacao_especial': 'special_situation_date',
+    'data_situacao': 'situation_date',
+    'efr': 'responsible_federative_entity',
+    'fantasia': 'trade_name',
+    'logradouro': 'address',
+    'motivo_situacao': 'situation_reason',
+    'municipio': 'city',
+    'natureza_juridica': 'legal_nature',
```

Comment:
What about `legal_entity`? Not sure about the context. Legal entity refers to the *subject* of a legal action. Legal nature refers to the sort of *object* (topic, issue, discussion, paradigm) related to law theory and philosophy. (Source: my mind — highly unreliable).